### PR TITLE
Small fix in previous FFN model. 

### PR DESCRIPTION
**Pull request by s1998**

### CODE REVIEW

**Comment by s1998** [comment]
@zoq  I was planning on writing the same for the CNN and CF. Should I go ahead ?

**Comment by zoq** [comment]
I think this is a great idea, please feel free to work on that.

File: Kaggle/DigitRecognizer/src/DigitRecognizer.cpp [hunk 1]
```diff
@@ -33,9 +33,9 @@ using namespace std;
 
 int main()
 {
-  // Dataset is randomly split into training 
-  // and validation parts with following ratio.
-  constexpr double RATIO = 0.9;
+  // Dataset is randomly split into validation 
+  // and training parts with following ratio.
+  constexpr double RATIO = 0.1;
```
Comment (zoq) [inline Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: Interesting, so it looks like the model is able to get good results even with a training set that is only 10 percent. Did you rerun the model with the updated ratio?

Comment (s1998) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: Yes I did. The results did go up.
Initially it was 79% and then it went up to 85%.

Comment (zoq) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: We are talking about the validation set right? This sounds similar https://github.com/mlpack/models/pull/5#pullrequestreview-94760466

Comment (s1998) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: Sorry, 85% was with LeNet, there was increase but I don't remember the exact number now :( .Sorry.

Comment (s1998) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: I ran the FNN model for 150 epochs : https://pastebin.com/hvLsF3M2
Best accuracy for validation set : 93.0952%

Comment (zoq) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: Okay thanks for the update, this looks good.

**Comment by zoq** [comment]
Can you open another PR for the LeNet?

File: Kaggle/DigitRecognizer/src/DigitRecognizer.cpp (hunk 1 above)
Comment (rcurtin) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: The value of `RATIO` here is actually the size of the test set, so it looks to me like it is 10% test set, 90% training set, not the other way around.

Comment (zoq) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: I guess, it would make sense to print the performance of the test set (extra file) ``testX`` at the end, to see the change in performance. I think it make sense to increase the training set size, if the performance of the test set increases.

Comment (s1998) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: The changes made are correct, right ?

**Comment by s1998** [comment]
Done.
https://github.com/mlpack/models/pull/9

File: Kaggle/DigitRecognizer/src/DigitRecognizer.cpp (hunk 1 above)
Comment (zoq) [reply Kaggle/DigitRecognizer/src/DigitRecognizer.cpp]: Sorry for the slow response on this one, I think they are correct, 90% training, 10% validation, and 100% test data. Perhaps we could see if validation 30% training 70% does make a huge difference. At the end this is an example, and somebody should tune the parameters if required.

**Comment by rcurtin** [comment]
Looks like this got forgotten about a little so I will go ahead and merge it now since it has been more than 3 days. :)
